Documentation/AdministratorManual/Templates/Start/Index.rst

```diff
@@ -63,9 +63,9 @@ the paths
 
     plugin.tx_daycarecenters {
       view {
-        templateRootPath = fileadmin/templates/ext/daycarecenters/Templates/
-        partialRootPath = fileadmin/templates/ext/daycarecenters/Partials/
-        layoutRootPath = fileadmin/templates/ext/daycarecenters/Layouts/
+        templateRootPath = EXT:daycarecenters/Resources/Private/Templates/
+        partialRootPath = EXT:daycarecenters/Resources/Private/Partials/
+        layoutRootPath = EXT:daycarecenters/Resources/Private/Layouts/
```

Comment:
You have written "You can use the following TypoScript in the **constants** to change
the paths", so I would prefer using `sitepackage` here

```suggestion
        templateRootPath = EXT:sitepackage/Resources/Private/Templates/
        partialRootPath = EXT:sitepackage/Resources/Private/Partials/
        layoutRootPath = EXT:sitepackage/Resources/Private/Layouts/
```